lib/reply.js

```diff
@@ -706,7 +714,11 @@ function sendStream (payload, res, reply) {
   } else {
     reply.log.warn('response will send, but you shouldn\'t use res.writeHead in stream mode')
   }
-  payload.pipe(res)
+  if (payload[kWebStreamReadable]) {
+    pipeline(payload, res, err => err && logStreamError(reply.log, err, res))
```

Comment:
is the callback right?

Maybe should have the same logic as the second eos call in this function.